CantoboardFramework/Keyboard/View/KeyCap.swift

```diff
@@ -245,10 +245,12 @@ indirect enum KeyCap: Equatable, ExpressibleByStringLiteral {
         case .rime(.delimiter, _, _): return "分"
         case .rime(.sym, _, _): return "符"
         case .reverseLookup(let schema): return schema.signChar
+        case .changeSchema(.yale): return "耶魯／劉錫祥"
         case .changeSchema(let schema): return schema.shortName
         case .toggleInputMode(.english, _): return "英文"
-        case .toggleInputMode(.chinese, let rimeSchema): return "中文 - \(rimeSchema?.shortName ?? "")"
-        case .toggleInputMode(.mixed, let rimeSchema): return "混合 - \(rimeSchema?.shortName ?? "")"
+        case .toggleInputMode(_, let rimeSchema): return rimeSchema?.shortName
+        case "'": return "′"
+        case "\"": return "″"
```

Comment:
This two lines handles the display of the apostrophes. I believe that you are going to handle whether the left or the right apostrophe is to be inserted.

Reply:
Could you revert this? This doesn't match the behaviour of stock iOS keyboard.